Stryker-cli parses missing install error incorrectly

When the stryker-cli can't run Stryker, it checks against the error `Cannot find module '@stryker-mutator/core'` using `indexOf` in the entire error.

In my case (Node v14.16.0) the entire error is: `Error: Cannot find module '@stryker-mutator/core/package.json' from 'C:\git\myrepo'`

So I believe a check for `Cannot find module '@stryker-mutator/core` without a single quote at the end would be more suited in this case.

--- a/src/stryker-cli.ts
+++ b/src/stryker-cli.ts
@@ -14,7 +14,7 @@ export function run(): Promise<void> {
   try {
     return runLocalStryker();
   } catch (error) {
-    if (error.toString().indexOf(`Cannot find module '@stryker-mutator/core'`) >= 0) {
+    if (error.toString().indexOf(`Cannot find module '@stryker-mutator/core`) >= 0) {
       return promptInstallStryker().then(packageManager => {
         if (packageManager !== undefined) {
           installStryker(installCommands[packageManager]);

